### PR TITLE
Make `display_list/...` compatible with `.clang_tidy`.

### PR DESCRIPTION
**Pull request by matanlurey**

### CODE REVIEW

**Comment by matanlurey** [comment]
Friendly ping @flar. + @jonahwilliams as well in case Jim is out.

File: display_list/benchmarking/dl_benchmarks.h [hunk 1]
```diff
@@ -571,16 +571,16 @@ void BM_SaveLayer(benchmark::State& state,
...
-                             kAntiAliasing_Flag)                         \
-  FILL_BENCHMARKS(BACKEND, kFilledStyle_Flag)                            \
-  FILL_BENCHMARKS(BACKEND, kFilledStyle_Flag | kAntiAliasing_Flag)       \
-  ANTI_ALIASING_BENCHMARKS(BACKEND, kEmpty_Flag)                         \
-  ANTI_ALIASING_BENCHMARKS(BACKEND, kAntiAliasing_Flag)                  \
-  OTHER_BENCHMARKS(BACKEND, kEmpty_Flag)
+  STROKE_BENCHMARKS(BACKEND, kStrokedStyle)                         \
+  STROKE_BENCHMARKS(BACKEND, kStrokedStyle | kAntiAliasing)    \
+  STROKE_BENCHMARKS(BACKEND, kStrokedStyle | kHairlineStroke)  \
+  STROKE_BENCHMARKS(BACKEND, kStrokedStyle| kHairlineStroke | \
```
Comment (flar) [inline display_list/benchmarking/dl_benchmarks.h:577]: Looks like a missing space before the `|` operator

**Review by flar — APPROVED** [review]
LGTM - other than the missing space

File: display_list/benchmarking/dl_benchmarks.h (hunk 1 above)
Comment (matanlurey) [reply display_list/benchmarking/dl_benchmarks.h:577]: Thanks. Done!

Comment (flar) [reply display_list/benchmarking/dl_benchmarks.h:577]: Oh, and the `\` at the ends of lines should line up. Also seen in other of these changes...

Comment (flar) [reply display_list/benchmarking/dl_benchmarks.h]: Sorry, removed the autosubmit pending fixing the alignment of the line ends...

Comment (matanlurey) [reply display_list/benchmarking/dl_benchmarks.h]: Should be fixed now.

**Pull request merged**